recipes/openfisca-france/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "OpenFisca-France" %}
+{% set version = "103.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/c4/55/7e186b3761f6abe6f502fe7be830a9a8f6721828dc6b919e7481d8486954/OpenFisca-France-{{ version }}.tar.gz
```

Comment:
Can you use `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/OpenFisca-France-{{ version }}.tar.gz` instead? That hashed URL will probably break the updating bot.

Reply:
Thanks, it's updated.